content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,200 @@
+We’ve improved the search UI to make the platform more intuitive with small usability improvements while making it easier for new users to get started with tips and recommendations on the revamped homepage. Performance across the platform saw a boost by using the latest in React, lazy loading, and priority rendering. These usability and performance enhancements help you get quick answers about your codebase and get unblocked faster. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Search]
+
+Docs: provide a link to the updated docs
+
+- Smarter code search
+### Understand usage and search structure with high-level aggregations of search results
+High-level aggregations of search results powered by Code Insights are now displayed in addition to results. Responses to the query are now grouped by location (repo or file), author, and arbitrary capture group pattern to help answer to questions such as:
+
+- Which files have the bulk of these search results?
+- Which repositories is this library used in most?
+- Which subcomponents are used most from this library?
+- Who is using our new innersource library?
+- Which areas of the code were added back in these regressions?
+- What are all the possible arguments we can pass to this function?
+- Aggregation can easily be reused by saving the visualization to your Code Insights dashboard.
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Insights]
+
+Docs: provide a link to the updated docs
+### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
+Code navigation allows you to quickly jump to symbol definitions, find references, dependencies, and more, so you can spend more time shipping code. Precise code navigation is the most powerful version of code navigation and it works cross-repository. We now support 9 languages including Ruby, Rust, Go, Java, Scala, Kotlin, Python, TypeScript, JavaScript. Auto-indexing makes precise code navigation work across all of your code, automatically, and it’s turned on by default for our Cloud customers. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Navigation]
+
+Docs: provide a link to the updated docs	
+
+### Access code context while staying within the developer workflow 
+Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, are now part of the core functionality. These enhancements provide developers code context while staying in the flow within their preferred tools. Additionally, the current Sourcegraph extensions framework will be deprecated so that we can make way for a new model that allows a deeper integration throughout our entire code intelligence platform. Please note, these changes do not impact our IDE extensions. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Integrations]
+
+Docs: provide a link to the updated docs	
```

Comment:
@ryankscott Please provide appropriate media and a link to the updated docs when they're available.